low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -192,6 +192,7 @@ SYSTEM_INFORMATION:
    environment: PROD
    sysfs_base_path: /sys/
    sspl_state: active
+   global_config_copy_url: ""
```

Comment:
why copy url path needed in config. If not seen in config, code can handle it right ?

Reply:
if global_config_copy_url(any key)  is not in default config file but it is in existing config. conf_upgrade script treat that as removed key. So, it will  not get added into existing config file. SSPL service failed because of this.
global_config_copy_url key get added later in provisioning steps.

Reply:
Ok so the value gets  updated during provisioning, then looks fine.